docs/02_cva6_requirements/cva6_requirements_specification.rst

```diff
@@ -140,15 +140,15 @@ The full list of parameters for this configuration will be detailed in the users
 
 Below are the configuration of the first releases of the CVA6.
 
-+--------------------+---------+---------+------+-------+---------+---------+---------+---------+
-| Release ID         | Target  | ISA     | XLEN | FPU   | CV-X-IF | MMU     | L1 D$   | L1 I$   |
-+====================+=========+=========+======+=======+=========+=========+=========+=========+
-| ``CV32A60X``       | ASIC    | IMC     |  32  | No    | Yes     | None    | 2 kB    | 2 kB    |
-+--------------------+---------+---------+------+-------+---------+---------+---------+---------+
-| ``CV32A60AX``      | ASIC    | IMC     |  32  | No    | Yes     | Sv32    | 16kB    | 16 kB   |
-+--------------------+---------+---------+------+-------+---------+---------+---------+---------+
-
-CV32A60X could evolve to CV32A65X if the team decides to integrate the dual-issue optional architectural feature.
++--------------------+---------+-----------------+------+-------+---------+---------+---------+---------+
+| Release ID         | Target  | ISA             | XLEN | FPU   | CV-X-IF | MMU     | L1 D$   | L1 I$   |
++====================+=========+=================+======+=======+=========+=========+=========+=========+
+| ``CV32A60X``       | ASIC    | IMC             |  32  | No    | Yes     | None    | 2 kB    | 2 kB    |
++--------------------+---------+-----------------+------+-------+---------+---------+---------+---------+
+| ``CV32A60AX``      | ASIC    | IMC             |  32  | No    | Yes     | Sv32    | 16kB    | 16 kB   |
++--------------------+---------+-----------------+------+-------+---------+---------+---------+---------+
+| ``CV64A60AX``      | ASIC    | IMACFDB_Zicount |  64  | Yes   | Yes     | Sv39    | 32kB    | 32 kB   |
```

Comment:
Did you mean `IMACFDB_Zicond` instead of `IMACFDB_Zicount`

Reply:
Upon reflection, I'll simplify to the main extensions and drop Zicount.
@fatimasaleem, can you check the main extensions for CV32A60AX?

Reply:
Looks good now! Thanks